client/src/SharedComponents/Slideshow/Slideshow.js

```diff
@@ -2,35 +2,47 @@ import React, { Component } from 'react';
 import StudentCard from '../StudentCard';
 import Slider from 'react-slick';
 import ProjectView from '../ProjectView';
+import { Link } from 'react-router-dom';
 import './Slideshow.css';
 
 class Slideshow extends Component {
   render() {
     const { projects, students, mentors } = this.props;
 
-    // Use new set proparity to remove redundent images of projects that comes from join query
-    const projectImages = projects.map((project) => {
-      return project.project_image;
-    });
-    let uniqueProjects = [...new Set(projectImages)];
-    const projectSlide = uniqueProjects.map((image) => {
-      return <ProjectView projectImg={image} />;
+    const projectSlide = projects.map((project) => {
```

Comment:
You could destructure the project right here, if you want:
`const projectSlide = projects.map(({project_id, project_image}) => {`